src/Apps/Order/Routes/Shipping2/Components/FulfillmentDetails.tsx

```diff
@@ -91,45 +91,6 @@ export const FulfillmentDetails: FC<FulfillmentDetailsProps> = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [hasSavedAddresses])
 
-  /*
-   * Re-save fulfillment details on load if they are already saved &
-   * require artsy shipping (new address form mode only - saved addresses
-   * handle this separately)
-   */
-  useEffect(() => {
-    const { savedFulfillmentDetails } = shippingContext.orderData
```

Comment:
Moved this to the outer Shipping2/index.tsx component since its concerns span beyond this file and it has no local dependencies. This greatly improves readability.